### ISSUE TITLE
"Prevent auto-play" and "DISABLE annotations" don't work... on this video

### ISSUE THREAD

**Issue by Fofer** [YePpHa/YouTubeCenter#1869]
I have "prevent auto-play" checked
and do NOT have "enable annotions" checked (I think this should be worded differently, actually, since checking the box really should flip the default state, like the other settings do)

and yet, the video still auto-plays, and I still see annotations.

On this Video: https://www.youtube.com/watch?v=ztiNeCKskWM

**Comment by Fofer** [comment]
It's strange and inconsistent.  Now it's auto-playing the first second or two, and then stopping.  Is there a way to actually prevent/block any auto-playing at all, in the foreground and background?

**Comment by DAOWAce** [comment]
There's numerous issues with the addon; it just keeps getting piled up over time as development seems to have very long breaks of updates.

I've had this issue opened for 7 months: https://github.com/YePpHa/YouTubeCenter/issues/1116  It was originally written to mention the flash player options being broken, but now HTML5 has been broken as well.

There's still not been any fix to prevent auto-play as far as I can see.

At this point, I might just have to use prevent auto-buffer..

**Comment by fohrums** [comment]
+1 for having `enable annotations` be worded differently.

@DAOWAce `...HTML5 has been broken as well.` I'm able to strictly use HTML5 without flashplayer now-a-days especially for YouTube, but auto-play isn't loading the video. In-fact, it seems as if it's stopped/haulted and therefore I have to manually switch to the video tab and do a play/pause for it to buffer as `without DASH enabled` is working properly.

**Comment by SuperSajuuk** [comment]
@DAOWAce I have to agree that development of the addon is so slow and getting to the point of unmaintainable. @YePpHa should seriously consider start giving some of the Collaborator users (such as @Yonezpt) the ability to make commits to the repository.

In fact, with the number of open tickets increasing every day and showing NO sign of slowing down, there will be no more "stable" build and everyone will end up waiting for developer updates.

Honestly, if updates don't start coming soon, this extension is going to end up being so broken that YouTube will just stop working. Whatever @YePpHa is doing, he needs to seriously consider providing more users commit access, instead of hogging it to himself.

**Comment by ngld** [comment]
@SuperSajuuk If you take a look at [GitHub's help page](https://help.github.com/articles/permission-levels-for-a-user-account-repository/), you'll see that collaborators already have commit permission. Also, most pull requests have been merged within 2 days.
I guess the real problem is that there aren't enough developers willing/able to fix these issues.

@YePpHa [has already found found a fix](https://github.com/YePpHa/YouTubeCenter/issues/1844#issuecomment-106532646). Apparently he just needs to release a new dev build.

**Comment by Yonezpt** [comment]
@ngld In my case I just am not comfortable with changing his code. Whenever I tried in the past it always broke something, if not everything, so I decided to just share my findings with the author and he'll apply them knowing exactly what can and can't be changed in his code.

**Comment by SuperSajuuk** [comment]
@Yonezpt Have you considered forking the repository code? That might make it easier for you to contribute code fixes, since I can't see the number of open tickets being reduced any time soon. I assume collaborators have the ability to close tickets? If not, they should, so a lot of these clearly duplicate tickets can be closed.

**Comment by Yonezpt** [comment]
@SuperSajuuk Forking wouldn't solve anything, the issues I faced with implementing my fixes on his code would still have the same outcome. Regarding handling issues, I've already considered doing that, but it might cause issues for the author since there are some issues which might contain valuable information to him which he won't have knowledge of if I close them myself, including duplicates because there are some occasions when they might appear duplicate to me, but the author may in fact notice that they are not the same.

**Comment by ireun** [comment]
@Fofer I've edited the first post to comply with "One issue per Github issue" rule. If "disable annotations is still not working" please create a new issue.

--
**This will now become official issue for all auto-play related issues.**
